packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -41,7 +51,11 @@ export default function shopifyGraphQLProxy(proxyOptions?: ProxyOptions) {
         'X-Shopify-Access-Token': accessToken,
       },
       proxyReqPathResolver() {
-        return `https://${shop}${GRAPHQL_PATH}`;
+        if (apiVersion) {
+          return `https://${shop}${GRAPHQL_PATH_PREFIX}${apiVersion}/${GRAPHQL_PATH_SUFFIX}`;
```

Comment:
Might want to just have `GRAPHQL_PATH_PREFIX` not have a trailing `/` since it's a bit unclear just from reading this bit what strings the path segments are part of (that is, explicitly put the path separators in between each part)